docs/docs-content/tutorials/getting-started/palette-edge/prepare-edge-host.md

```diff
@@ -0,0 +1,197 @@
+
+Power on the device and enter the Basic Input/Output System (BIOS) interface. You can accomplish this by pressing
+**F2**, **F1**, or **F10** immediately after powering on the device. The exact key varies by manufacturer. Consult the
+manufacturer instructions to learn how to enter the BIOS interface.
+
+In the BIOS interface, navigate to the boot sequence section and locate the boot sequence list.
+
+Find the entry with your USB drive and move it to the top of the list. Save the changes and exit the BIOS interface. The
+device then boots from the USB drive and begins installing the Palette agent.
+
+Wait for the Edge Installer to complete copying content to the device. This process may take a few minutes. When the
+installation is complete, the device shuts down automatically. This behavior is configured in the `user-data` file, as
+specified in the [Prepare User Data](./prepare-user-data.md) tutorial with the line `poweroff: true`.
+
+Once the device powers off, remove the USB drive. Since it was previously selected as the boot volume, leaving it
+inserted would cause the system to boot from it again, restarting the installation process.
+
+The device is now ready to be registered with Palette as an Edge host.
+
+</TabItem>
+
+</Tabs>
+
+## Register Edge Host
+
+<Tabs groupId="host">
+
+<TabItem label="VM Host" value="VM Host">
+
+To register the host with Palette, select the VM in VirtualBox and click **Start** to power it on. The VM boots and
+obtains an IP address from the host machine's bridged network.
+
+After a few minutes, the VM screen displays an IP address and automatically registers with Palette as an Edge host,
+using the registration token provided in the `user-data` file during the
+[Prepare User Data for Edge Installation](./prepare-user-data.md) tutorial.
+
+![A screenshot of the Edge host.](/getting-started/getting-started_introduction-edge_prepare-edge-host_edge-host-screen.webp)
+
+:::tip
+
+You can provide site-specific Edge installer configuration user data if you need to apply new values or override default
+values from the Edge installer user data you created during the EdgeForge process. Refer to
+[Apply Site User Data](../../../clusters/edge/site-deployment/site-installation/site-user-data.md) for more information.
+
+:::
+
+Log in to [Palette](https://console.spectrocloud.com/). From the left main menu, select **Clusters**. Click the **Edge
+Hosts** tab to view the registered hosts.
+
+Confirm that your Edge host is listed with a **Healthy** and **Ready** status. The **Machine ID** displayed in Palette
+should match the ID displayed on the VM screen.
+
+![A screenshot of the Edge host in Palette.](/getting-started/getting-started_introduction-edge_prepare-edge-host_edge-host-palette.webp)
+
+</TabItem>
+
+<TabItem label="Bare Metal Host" value="Bare Metal Host">
+
+To register the host with Palette, power on your bare metal device. Let Palette Edge choose the registration boot option
```

Comment:
Did we already mention that the USB needs to be plugged in? I know it's obvious but I may have missed it. Also once the installation is complete they have to unplug the USB.